AttitudeManager/FlightModes/Src/fbwa.cpp

```diff
@@ -0,0 +1,38 @@
+#include "fbwa.hpp"
+#include "CommonDataTypes.hpp"
+#include "AM.hpp"
+#include "config.hpp"
+#include <cassert>
+
+namespace AM {
+    AttitudeManagerInput FBWA::run(const AttitudeManagerInput& input) {
+        
+        AttitudeManagerInput mappedOutputs;
+
+        assert(AM::AttitudeManager::INPUT_MAX == 100 && AM::AttitudeManager::INPUT_MIN == -100);
+
+        //Importing max/min limits here
+        float yawMin = config::flightmodes[1].tuningData.controlLimits.yawLimit.min;
+        float yawMax = config::flightmodes[1].tuningData.controlLimits.yawLimit.max;
+
+        float rollMin = config::flightmodes[1].tuningData.controlLimits.rollLimit.min;
+        float rollMax = config::flightmodes[1].tuningData.controlLimits.rollLimit.max;
+        
+        float pitchMin = config::flightmodes[1].tuningData.controlLimits.pitchLimit.min;
+        float pitchMax = config::flightmodes[1].tuningData.controlLimits.pitchLimit.max;
+
+        // Mapping inputs to outputs with ratios of min-max
+        mappedOutputs.pitch = pitchMin + (input.pitch - AM::AttitudeManager::INPUT_MIN) * (pitchMax - pitchMin) / (AM::AttitudeManager::INPUT_MAX - AM::AttitudeManager::INPUT_MIN);
+        mappedOutputs.roll = rollMin + (input.roll - AM::AttitudeManager::INPUT_MIN) * (rollMax - rollMin) / (AM::AttitudeManager::INPUT_MAX - AM::AttitudeManager::INPUT_MIN);
+        mappedOutputs.yaw = yawMin + (input.yaw - AM::AttitudeManager::INPUT_MIN) * (yawMax - yawMin) / (AM::AttitudeManager::INPUT_MAX - AM::AttitudeManager::INPUT_MIN);
+        mappedOutputs.throttle = input.throttle;
```

Comment:
We'll also need to scale this, because your input will be -100 -> 100, and we need that to map to 0 -> 100 I think?

This depends a little bit on @csabahi 's expected PID (?)

Reply:
Just confirmed - we are expecting throttle outputs to be 0-100, so we have the option of either running a PID with setpoint == input, and current pos == input, or we can remap throttle to be 0-100.